### ISSUE TITLE
Cannot access Google drive

### ISSUE THREAD

**Issue by tnnandi** [glotlabs/gdrive#71]
Firstly, I'd like to thank you for the immense effort you put into this tool and explaining to users the 50 steps in a very detailed manner to create Google API credentials https://github.com/glotlabs/gdrive/blob/main/docs/create_google_api_credentials.md (those are some of the best set of instructions I've ever seen).

Now coming to the problem: after setting up all the credentials, then trying to connect to Google drive (using Chrome/Edge browser) using the link provided after executing "./gdrive account add" on a linux terminal, and finally getting past the step where I choose the gmail id that I'd like to connect, I end up seeing the following error (the screenshot is attached too):

Hmmm… can't reach this page127.0.0.1 refused to connect.
Try:

Search Bing for [127 0 0 1](https://www.bing.com/search?form=ANLKDR&q=127%200%200%201)

Checking the connection

[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)

ERR_CONNECTION_REFUSED

![Capture](https://github.com/glotlabs/gdrive/assets/12446612/789c328d-c3b4-4ff4-ba9c-c707c5a9867a)

**Comment by mcassatt** [comment]
I am having the same issue, did you ever figure out how to solve this @tnnandi ?

**Comment by rushatrai** [comment]
Did you manage to solve this?

**Comment by 3rdglaz** [comment]
Did you try getting to the port 8085?

> http://127.0.0.1:8085

**Comment by kaishxu** [comment]
> Did you try getting to the port 8085?
> 
> > http://127.0.0.1:8085

I encountered the same issue.

**Comment by 3rdglaz** [comment]
Are you trying to configure the dgrive add on a remote machine?

**Comment by TheSashmo** [comment]
The way I solved this was installing gdrive on my mac, then doing the whole process there, then copying the config folder from my mac over to my linux box, and  it worked perfectly fine, just keep the same location of where you put that token.

**Comment by SiyiSEA** [comment]
I have the same issue. Anyone has solved it based on the linux in Windows? Thanks.

**Comment by dfguan** [comment]
yeah, me too. I was trying to set up the drive on a remote machine. If anyone resolved it, please do tell.

**Comment by dfguan** [comment]
I have got this solution from a discord discussion, turns out we need to first build connection on our local machine, then use ``gdrive account export <account_name>`` to export connection information and upload to the remote machine and run ``gdrive account import <file_name>``. It works well.
